Jenkinsfile

```diff
@@ -335,6 +356,99 @@ az account set -s $AZURE_SUBSCRIPTION_ID
         }
     }
 
+    // See:
+    //  gcloud container get-server-config
+    def gkeKversions = ["1.9", "1.10"]
+    for (x in gkeKversions) {
+        def kversion = x  // local bind required because closures
+        def platform = "gke-" + kversion
+        platforms[platform] = {
+            stage(platform) {
+                node(label) {
+                    withCredentials([
+                        file(credentialsId: 'gke-kubeprod-jenkins', variable: 'GOOGLE_APPLICATION_CREDENTIALS'),
+                        usernamePassword(credentialsId: 'gke-oauthproxy-client', usernameVariable: 'OAUTH_CLIENT_ID', passwordVariable: 'OAUTH_CLIENT_SECRET'),
+                    ]) {
+                        withEnv(["CLOUDSDK_CORE_DISABLE_PROMPTS=1"]) {
+                            withGo() {
+                                dir('src/github.com/bitnami/kube-prod-runtime') {
+                                    def project = 'bkprtesting'
+                                    def zone = 'us-east1-d'
+                                    def clusterName = ("${env.BRANCH_NAME}".take(8) + "-${env.BUILD_NUMBER}-" + UUID.randomUUID().toString().take(5) + "-${platform}").replaceAll(/[^a-zA-Z0-9-]/, '-').toLowerCase()
+                                    def dnsPrefix = "${clusterName}"
+                                    def parentZone = 'tests.bkpr.run'
+                                    def adminEmail = "${clusterName}@${parentZone}"
+                                    def dnsZone = "${dnsPrefix}.${parentZone}"
+
+                                    try {
+                                        runIntegrationTest(platform, "gke --project=${project} --dns-zone=${dnsZone} --email=${adminEmail} --authz-domain=\"*\"", "") { // --dns-suffix ${dnsZone}
+                                            container('gcloud') {
+                                                sh "gcloud auth activate-service-account --key-file ${GOOGLE_APPLICATION_CREDENTIALS}"
+                                                sh """
+gcloud container clusters create ${clusterName} \
+ --cluster-version ${kversion} \
+ --project ${project} \
+ --create-subnetwork '' \
+ --enable-ip-alias \
```

Comment:
I think we should remove `--create-subnetwork "" --enable-ip-alias` ?
I can't remember why I put them there (anticipating one of the many future-compatibility-warnings that `gcloud c c create` reports?), and they aren't mentioned in the gke quickstart.  Better to test whatever we recommend in the docs.

Reply:
yep I will remove this. It's not needed

Reply:
cleaned up